src/libs/localStore/storage.ts

```diff
@@ -24,21 +28,35 @@ export enum StorageKeys {
   GeolocationTrackingConfig = 'CozyGPSMemory.TrackingConfig',
   Activities = 'CozyGPSMemory.Activities',
   FirstTimeserie = 'CozyGPSMemory.FirstTimeserie',
-  ServiceWebhookURL = 'CozyGPSMemory.ServiceWebhookURL'
+  ServiceWebhookURL = 'CozyGPSMemory.ServiceWebhookURL',
+  Oauth = '@cozy_AmiralAppOAuthConfig',
+  Cookie = '@cozy_AmiralAppCookieConfig',
+  SessionCreated = '@cozy_AmiralAppSessionCreated'
+}
+
+/*
+  Linked to device.
+  Not removed at logout.
+*/
+export enum DevicePersistedStorageKeys {
+  OnboardingPartner = '@cozy_AmiralAppOnboardingPartnerConfig',
+  ClouderyEnv = '@cozy_AmiralAppClouderyEnvConfig',
+  Bundle = '@cozy_AmiralAppBundleConfig'
 }
 
 export type IconsCache = Record<string, { version: string; xml: string }>
 
 export interface StorageItems {
   biometryActivated: boolean
   biometryType?: BiometryType
-  sessionCreatedFlag: string
   iconCache: IconsCache
   firstTimeserie: FirstTimeserie
+  onboardingPartner: object
```

Comment:
What is the reason for not using the `OnboardingPartner` type here? Is that to prevent cyclic imports?

Reply:
To prevent cyclic imports and because I was a bit lazy too. `import type { ... }` does not show a cyclic import at the start of the app so I updated the PR with it. I also improved other types in the meatime.